### PR TITLE
Fixes IE11 dropdown position #1733

### PR DESCRIPTION
**Pull request by toni-sharpe**
# Description

* `window.scrollY` is not supported by IE11 and below so it's replaced with the more comprehensive `window.pageYOffset`.
* https://stackoverflow.com/questions/44757869/window-pageyoffset-vs-window-scrolly-on-ie11.

# TODO
- [x] Release notes - edited to refer initial fix to issue

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # 3.1.4
 
-Fixes incorrect dropdown placement in IE.
+Fixes incorrect dropdown placement in IE. #1733
```
Comment (SeanArmstrong) [inline CHANGELOG.md]: I think we will need a new entry in the changelog for v3.1.5 and to update the package.json version to 3.1.5 and finally do a npm install. Otherwise we will have to do it in a separate PR

**Comment by SeanArmstrong** [comment]
Checked in Demo Site IE, EDGE, Safari, Chrome, Firefox

Checked in GAC via s1_gac_ui in IE, Edge, Safari, Chrome, Firefox

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
+# 3.1.5
+
+Fixes incorrect dropdown placement in IE11 #1733
+
 # 3.1.4
 
-Fixes incorrect dropdown placement in IE.
+Fixes incorrect dropdown placement in Edge #1733
```
Comment (ianoxley) [inline CHANGELOG.md]: Not sure we should change release notes for already released versions. Maybe just highlight the fact the previous release only fixed the issue in Edge in the current versions release notes?

Also can you link the issue number to the issue please?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # 3.1.5
 
-Fixes incorrect dropdown placement in IE11 #1733
+Fixes incorrect dropdown placement in IE11. v3.1.4 only fixed Edge #1733
 
 # 3.1.4
 
-Fixes incorrect dropdown placement in Edge #1733
+Fixes incorrect dropdown placement in IE.
```
Comment (toni-leigh) [inline CHANGELOG.md]: This isn't actually correct, it fixes it in Edge, not IE
